examples/examples-gpu/nyc-taxi/rf-rapids-dask.ipynb

```diff
@@ -4,9 +4,7 @@
    "cell_type": "markdown",
```

Comment:
<p>Pretty cool indeed!!</p><p> </p><p>also <code>diagnostic</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/saturncloud/examples/pull/58/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='39'/>

Reply:
<p>no no it's diganostic, it's a new type of experimental jazz</p>

Reply:
<p>fixed in https://github.com/saturncloud/examples/pull/58/commits/58a8c4dcce6b6821e95ff31d5bbd90a9e4abce46</p>